src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,33 @@
-export const PostInfo = () => (
-  <>Put the post here</>
-);
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+import './PostInfo.scss';
+
+export const PostInfo = ({ post }) => {
+  const {
+    title,
+    body,
+    user,
+    comments,
+  } = post;
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{title}</h3>
+
+        <UserInfo user={user} />
+      </div>
+
+      <p className="PostInfo__body">
+        {body}
+      </p>
+
+      <hr />
+
+      {comments.length > 0
+        ? <CommentList comments={comments} />
+        : <b data-cy="NoCommentsMessage">No comments yet</b>
+      }
```

Comment:
need to use `<hr />` only if there are no comments

Reply:
> need to use `<hr />` only if there are no comments

can't find requirement about it

Reply:
> 
in the initial file only when there are no comments - there is `<hr />`
otherwise - none (see screenshots below)


No comments:
--------------------------------------------------------
![image_2023-08-23_14-53-13](https://github.com/mate-academy/react_static-list-of-posts-js/assets/91376823/0b0e1758-1a01-49bf-85c5-f2e7e74d688e)
Has comments:
--------------------------------------------------------
![image_2023-08-23_14-53-13 (2)](https://github.com/mate-academy/react_static-list-of-posts-js/assets/91376823/5d3582c8-a8a3-4be3-8926-fd969408435d)